src/cockpit/channels/packages.py

```diff
@@ -53,18 +53,11 @@ def do_done(self) -> None:
         options = self.options
 
         try:
-            if options.get('method') != 'GET':
+            if get_str(options, 'method') != 'GET':
                 raise ValueError(f'Unsupported HTTP method {options["method"]}')
 
-            path = options.get('path')
-            if not isinstance(path, str) or not path.startswith('/'):
-                raise ValueError(f'Unsupported HTTP method {options["method"]}')
```

Comment:
The startswith check is still relevant, and should come back. (Obviously the old error message was a bad copy-pasta)

Reply:
I think we verify that inside of `cockpit.packages` but I can check.

Reply:
We do it there as an assert, which is a lot less nice.  We also don't deal nicely with the case where we have a path like `/apps`.  I'll rework that a bit so that both of those turn into 404s.

Reply:
My thinking: the function takes a parameter which has the static type `str`.  It should work with all `str`, and not rely on the `/` check being done outside (effectively making the static type be `str`-starting-with-`/`).